resources/public/api/conf/2.0/fhl_uk_property_business_annual_submission_create_and_amend.raml

```diff
@@ -0,0 +1,37 @@
+is:
+  - headers.acceptJson
+  - headers.contentTypeJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.ruleTaxYearRangeInvalid
+  - errors.formatTaxYear
+  - errors.ruleTaxYearNotSupported
+  - errors.formatFhlValue
+  - errors.ruleIncorrectOrEmptyBody
+  - errors.clientOrAgentNotAuthorised
+  - errors.notFound
```

Comment:
I wonder if it is worth confirming with a B.A that we are meant to include this error here? The "matching-resource" error is included in the tech spec but is not in the business spec.

Reply:
The Business requirement section has now be updated to be in-line with tech requirement